index.html

```diff
@@ -114,7 +114,50 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+	      
+	      
 	 
+	      <!-- ________ *MOLLY'S card START ________  -->
+	      <div class="card">
+		      <p class="name">MOLLY VANESSA</p>
+		      <p class="contact">
+            <a href="https://twitter.com/molly_simply" target="_blank">@molly_simply
+		        <a href="https://github.com/molly460" target="_blank"><i class="fab fa-github"></i></a>
+		        <a href="https://www.linkedin.com/in/mollyvanessa/" target="_blank"><i class="fab fa-linkedin"></i></a>
+		      </p>
+		      <p class="about">I'm a front end developer willing to contribute to open source projects.
+            I'm passionate about coding, love travelling and photography.
+          </p>
+          <div class="skill">
+           <h3>Tech Skills</h3>
```

Comment:
Creating a `<h3>` tag in the middle of the `<div>` here is okay. But it will make the card look ugly. This project contains CSS for the original template. 
Please check it out and get back to us.